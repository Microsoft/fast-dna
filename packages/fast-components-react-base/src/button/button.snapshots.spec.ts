// import examples from "./examples.data";
// import {generateSnapshots} from "../../../../test/generate-snapshots";

// generateSnapshots(examples);
