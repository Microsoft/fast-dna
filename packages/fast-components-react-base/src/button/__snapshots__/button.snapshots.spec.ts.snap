// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button: 0 1`] = `
<button
  className="button"
>
  Button text
</button>
`;

exports[`button: 1 1`] = `
<button
  className="button-2"
>
  Button text foo
</button>
`;
